style.css

```diff
@@ -0,0 +1,990 @@
+
+@media (max-width: 860px) {
+  .text__wrap_main,
+  .text__wrap_character { padding-left: 9px; }
+}
+
+/* Планшет */
+@media (max-width: 768px) {
+  .section-desc {
+    display: block;
+    width: auto;
+    height: auto;
+    background: 0;
+  }
+
+  .section-desc__wrap_up {
+    height: 486px;
+    padding-top: 45px;
+    background: url(./Project/item_interior.svg) no-repeat;
+    background-size: contain;
+    background-position: 0 100%;
+  }
+
+  .section-desc__title {
+    width: 94%;
+    text-align: right;
+    padding-left: 0;
+  }
+
+  .text {
+    display: flex;
+    flex-wrap: wrap;
+    width: auto;
+    height: 485px;
+  }
+
+  .text__wrap_main {
+    width: 45.5%;
+    margin: 0;
+    padding-left: 33px;
+  }
+
+  .text__wrap_title {
+    margin-bottom: 17px;
+    padding-top: 54px;
+  }
+
+  .text__content {
+    width: 90%;
+    height: 220px;
+    margin: 0;
+    padding-left: 2px;
+    letter-spacing: .6px;
+  }
+
+  .text__wrap_character {
+    flex-direction: column;
+    margin: 0;
+    padding-top: 54px;
+    padding-left: 69px;
+  }
+
+  .text__wrap_character-up {
+    height: 248px;
+  }
+
+  .text__item_bag,
+  .text__item_small-wifi { padding-bottom: 0; }
+
+  .text__item_small-screen,
+  .text__item_small-bag { padding: 10px 0 0 2px; }
+  .text__item_small-screen { margin-right: 20px; }
+  .text__item_small-bag {  margin-right: 25px; }
+
+  .text__item_icon { max-width: 100%; }
+
+  .text__item_small-wifi { margin-right: 17px; }
+
+  .button {
+    align-items: center;
+    width: 70%;
+    margin: 0 auto;
+    padding-bottom: 7px;
+  }
+
+  .button__item_big-info { padding-left: 17px; }
+
+  .button__item_icon {
+    width: 30px;
+    margin-right: 11px;
+  }
+
+  .button__item_big-right {
+    width: 41%;
+    padding-right: 17px;
+  }
+  .button__item_big-order { padding-left: 0; }
+}
+
+@media (max-width: 650px) {
+  .section-desc__wrap_up { padding-top: 0; }
+
+  .text__wrap_main {
+    width: 48%;
+    padding-left: 20px;
+  }
+  
+  .text__item_icon { max-width: 90%; }
+
+  .button__item_icon { width: auto; }
+}
+
+@media (max-width: 560px) {
+  .section-desc__wrap_little { height: 385px; }
+  
+  .text__wrap_main { padding-left: 12px; }
+
+  .text__title { letter-spacing: 0; }
+
+  .text__item_small-screen,
+  .text__item_small-bag {
+    margin-right: 10px;
+    padding-left: 4px;
+  }
+
+  .text__item_small-wifi { margin-right: 5px; }
+
+  .button { width: 90%; }
+
+  .button__item_big-right {
+    width: 30%;
+    height: 42px;
+  }
+}
+
+@media (max-width: 500px) {
+  .text__item_icon { max-width: 80%; }
+}
+
+@media (max-width: 445px) {
+  .section-desc__wrap_little { height: 322px; }
+}
+
+/* Телефон */
+@media (max-width: 375px) {
+  .section-desc { height: 626px; }
+  .section-desc__wrap_up {
+    background-size: cover;
+    background-position: -35px 5px;
+  }
+
+  .section-desc__title,
+  .text__wrap_character,
+  .button { display: none; }
+
+  .text { height: 305px; }
+
+  .text__wrap_main { 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Project_page/pullRequest?prid=5536520)